doc/release-process.md

```diff
@@ -43,6 +43,11 @@ Release Process
     - On mainnet, the selected value must not be orphaned, so it may be useful to set the height two blocks back from the tip.
     - Testnet should be set with a height some tens of thousands back from the tip, due to reorgs there.
   - `nMinimumChainWork` with the "chainwork" value of RPC `getblockheader` using the same height as that selected for the previous step.
+  - `m_assumeutxo_data` with the updated assumeutxo hash and nChainTx count.
+    - You can obtain this information, and the corresponding snapshot, by running
+      `./contrib/devtools/utxo_snapshot.sh <blockheight> <snapshot-out-path>`.
+    - The height used should probably the be same as the assumevalid height chosen.
+    - Ensure the resulting snapshot is uploaded somewhere publicly accessible (torrent, HTTP server, etc.).
```

Comment:
In commit "doc: update release-process.md for assumeutxo" (27adc36a173d5636e82dfca27364dbda34f3b022)

This seems a little too vague for release instructions. Maybe this can be made more specific, or just marked as optional / TBD to distinguish it from the more well defined parts of the release process.

Reply:
Reworded.